### ISSUE TITLE
Updated landing page

### ISSUE THREAD

**Issue by kristinschrader** [glos/myglos#197]
Design a GLOS landing page that comes with pre-set views. Users can choose one of those or go into the catalog or map view. Example attached. 

[data portal landing page.pdf](https://github.com/glos/myglos/files/1759307/data.portal.landing.page.pdf)

**Comment by kknee** [comment]
love this idea - many portals going this way, e.g:

https://www.northeastoceandata.org/

**Comment by kristinschrader** [comment]
Squeal! Seriously I am updating adwords, and I'd much prefer to send people
to this than just the portal. I can send them to the web page, but it's not
necessarily true to the click through. Especially if I can send them *to*
one of those views. I'm not sure if we would generate a seperate address or
not.

Kristin Schrader
Great Lakes Observing System
734-741-2294
glos.us <http://www.glos.us>


Like our Facebook page
<https://www.facebook.com/GreatLakesObservingSystem/?ref=aymt_homepage_panel>
Follow our Twitter feed <https://twitter.com/GLOSystem>

**Comment by ColinNBrooks** [comment]
I agree - I think we need a "friendlier" entry page for the data portal,
which should help with user engagement & sending them to what they may be
looking for. The Northeast portal seems promising. - Colin


On Mar 15, 2018 1:42 PM, "Kelly Knee" <notifications@github.com> wrote:

love this idea - many portals going this way, e.g:

https://www.northeastoceandata.org/

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/glos/myglos/issues/197#issuecomment-373463233>, or mute
the thread

**Comment by kkoch** [comment]
I really like that northeast website.  It is very user friendly and has all the possibilities on the front page.
